File: InventoryAppOriginal/app/src/main/java/com/example/brookesellerinventoryapp/TabsAdapter.java
package com.example.brookesellerinventoryapp;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.viewpager2.adapter.FragmentStateAdapter;

public class TabsAdapter extends FragmentStateAdapter {

    public TabsAdapter(@NonNull FragmentActivity fa) {
        super(fa);
    }

    @NonNull
    @Override
    public Fragment createFragment(int position) {
        // position 0 = Inventory tab, position 1 = Passwords tab
        return position == 0 ? new InventoryFragment() : new PasswordsFragment();
    }

    @Override
    public int getItemCount() {
        return 2;
    }
}

File: BrookeSellerInventoryApp/app/src/main/kotlin/com/example/brookesellerinventoryapp/TabsAdapter.kt
package com.example.brookesellerinventoryapp

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentActivity
import androidx.viewpager2.adapter.FragmentStateAdapter

class TabsAdapter(fa: FragmentActivity) : FragmentStateAdapter(fa) {
    override fun createFragment(position: Int): Fragment {
        // position 0 = Inventory tab, position 1 = Passwords tab
        return if (position == 0) InventoryFragment() else PasswordsFragment()
    }

    override fun getItemCount(): Int {
        return 2
    }
}
